apps/onboarding/src/gateway/gateway.service.spec.ts

```diff
@@ -15,4 +19,24 @@ describe('GatewayService', () => {
   it('should be defined', () => {
     expect(service).toBeDefined()
   })
+
+  xit('should enable the rules depending on env variables', async () => {
```

Comment:
Remove the `x` it's causing tests to be skipped.
Also I think I'd rather test the full rule logic here. So instead of casting to `any` let's actually run a `.verify` and then spy on the rules themselves.
So basically we could call verify and then expect the `verify` method to be called on some rules and not the others.

Reply:
Before testing the verify method I would like to ensure that the config is being set successfully. So I think that are different tests.

Reply:
Yeah we can have both